modules/installation-osp-accessing-api-no-floating.adoc

```diff
@@ -3,10 +3,88 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-kuryr"]
+:osp-kuryr:
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user-kuryr"]
+:osp-kuryr:
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-restricted"]
+:osp-ipi:
+:osp-restricted:
+endif::[]
 
 [id="installation-osp-accessing-api-no-floating_{context}"]
-= Enabling access without floating IP addresses
+= Completing installation without floating IP addresses
+
+You can install {product-title} on {rh-openstack-first} without providing floating IP addresses.
+
+In the
+ifdef::osp-ipi[`install-config.yaml`]
+ifdef::osp-upi[`inventory.yaml`]
+file, leave the following
+ifdef::osp-ipi[parameters]
+ifdef::osp-upi[variables]
+blank:
+
+ifdef::osp-ipi[]
+* `platform.openstack.ingressFloatingIP`
+* `platform.openstack.lbFloatingIP`
+
+If you cannot provide an external network, you can also leave `platform.openstack.externalNetwork` blank.  If you do not provide a value for `platform.openstack.externalNetwork`, a router is not created for you, and, without additional action, the installer will fail to retrieve an image from Glance. You must configure external connectivity on your own.
+endif::osp-ipi[]
+
+ifdef::osp-upi[]
+* `os_api_fip`
+* `os_bootstrap_fip`
+* `os_ingress_fip`
+
+If you cannot provide an external network, you can also leave `os_external_network` blank. If you do not provide a value for `os_external_network`, a router is not created for you, and, without additional action, the installer will fail to retrieve an image from Glance. Later in the installation process, when you create network resources, you must configure external connectivity on your own.
+endif::osp-upi[]
+
+If you run the installer
+ifdef::osp-upi[with the `wait-for` command]
```

Comment:
So UPI installation succeeds if you don't use the `wait-for` command?

Reply:
Sometimes.